src/stylesheets/bootstrap_manifest.scss

```diff
@@ -0,0 +1,43 @@
+@import '~bootstrap/scss/functions';
```

Comment:
@rafayet-monon Should we omit this `~bootstrap/scss` out? 🤔

Reply:
As I am importing the modules from `node_modules`, omitting that `~bootstrap/scss` path will result in failure to import the module. Is there any better way that I am missing?